rubocop.yml

```diff
@@ -106,3 +108,13 @@ Style/SignalException:
 
 Layout/SpaceAroundOperators:
   AllowForAlignment: true
+
+Metrics/ModuleLength:
+  Exclude:
+    - "**/*_spec.rb"
+
+RSpec/MultipleExpectations:
+  Max: 10
+
+RSpec/NestedGroups:
+  Max: 6
```

Comment:
Same here: might as well turn it off: if anyone tries to write a spec with seven levels of nesting, it should look suspicious (like a symptom of a possible code smell) either while writing or in code review

Reply:
Turning it off is fine for me